projects/ngx-location-viewer/src/lib/components/layer-management/layer-management.component.html

```diff
@@ -1,19 +1,21 @@
 <h1 class="has-base-font u-text-bold h5">Lagen</h1>
 <div class="u-margin-left-xs">
     <div *ngIf="operationalLayer" class="u-margin-top">
-        <h2 class="has-base-font u-text-bold h6">Operationele laag:</h2>
+        <h2 class="has-base-font u-text-bold h6">{{ translations.operationalLayer }}</h2>
         <aui-layer
             *ngIf="operationalLayer"
+            [translations]="translations"
             [layer]="operationalLayer"
             [layerType]="layerTypes.operational"
             (layerVisibiltyChange)="onLayerVisibilityChange($event)"
         ></aui-layer>
     </div>
     <div *ngIf="supportingLayers" class="u-margin-top">
-        <h2 class="has-base-font u-text-bold h6">Ondersteunende lagen:</h2>
+        <h2 class="has-base-font u-text-bold h6">{{ translations.supportingLayers }}</h2>
```

Comment:
in een van deze vertalingen is het ":" meegenomen in het andere niet. Misschien best conistentie (ik zou persoonlijk de ":" niet meer in de vertalingen zetten maar consistentie is wat ik zoek

Reply:
Fixed!